core/domain/suggestion_services.py

```diff
@@ -2484,10 +2484,171 @@ def _update_question_review_stats_models(
+) -> None:
+    """Updates TranslationSubmitterTotalContributionStats
+    model for given translation submitter stats.
+
+    Args:
+        translation_submitter_total_stats:
+            TranslationSubmitterTotalContributionStats.
+            TranslationSubmitterTotalContributionStats domain object.
+
+    Raises:
+        Exception. Language is None.
+        Exception. Contributor user ID is None.
+    """
+    stat = translation_submitter_total_stats
+
+    stats_model = suggestion_models.TranslationSubmitterTotalContributionStatsModel.get( # pylint: disable=line-too-long
+            stat.language_code,
+            stat.contributor_id)
+
+    assert stats_model is not None
+    stats_model.topic_ids_with_translation_submissions = (
+        stat.topic_ids_with_translation_submissions)
+    stats_model.recent_review_outcomes = stat.recent_review_outcomes
+    stats_model.recent_performance = stat.recent_performance
+    stats_model.overall_accuracy = stat.overall_accuracy
+    stats_model.submitted_translations_count = (
+        stat.submitted_translations_count)
+    stats_model.submitted_translation_word_count = (
+        stat.submitted_translation_word_count)
+    stats_model.accepted_translations_count = (
+        stat.accepted_translations_count)
+    stats_model.accepted_translations_without_reviewer_edits_count = (
+        stat.accepted_translations_without_reviewer_edits_count)
+    stats_model.accepted_translation_word_count = (
+        stat.accepted_translation_word_count)
+    stats_model.rejected_translations_count = (
+        stat.rejected_translations_count)
+    stats_model.rejected_translation_word_count = (
+        stat.rejected_translation_word_count)
+    stats_model.first_contribution_date = stat.first_contribution_date
+    stats_model.last_contribution_date = stat.last_contribution_date
+
+    suggestion_models.TranslationSubmitterTotalContributionStatsModel.update_timestamps( # pylint: disable=line-too-long
+        stats_model,
+        update_last_updated_time=True)
+    suggestion_models.TranslationSubmitterTotalContributionStatsModel.put(
+        stats_model)
+
+
+def _update_translation_reviewer_total_stats_models(
+    translation_reviewer_total_stat:
+        suggestion_registry.TranslationReviewerTotalContributionStats
+) -> None:
+    """Updates TranslationReviewerTotalContributionStats
+    models for given translation review stats.
+
+    Args:
+        translation_reviewer_total_stat:
+            TranslationReviewerTotalContributionStats.
+            TranslationReviewerTotalContributionStats domain object.
+    """
+    stat = translation_reviewer_total_stat
+
+    stats_model = suggestion_models.TranslationReviewerTotalContributionStatsModel.get( # pylint: disable=line-too-long
+        translation_reviewer_total_stat.language_code,
+        translation_reviewer_total_stat.contributor_id)
+
+    assert stats_model is not None
```

Comment:
I guess we don't need to do this because we are calling this method only when the model exists. If model doesn't exist we create a new model in `update_translation_review_stats`

Reply:
Updated with a comment. PTAL.